### DISCUSSION TITLE
Game stutters when song starts playing

### DISCUSSION THREAD

**Discussion by pszer** [ppy/osu#21068] [Q&A] [answered]
### Type

Performance

### Bug description

The game stutters when a song's audio starts playing in both the editor and in gameplay, I'm using an audio offset of 127ms in the video so the stutter happens a bit after the first note.

This is on Linux.

### Screenshots or videos

https://user-images.githubusercontent.com/44235930/199369233-68d62fbc-7b22-4cc7-beaf-2ea96628d530.mp4

https://user-images.githubusercontent.com/44235930/199369241-7f7fb6e1-32a3-4604-a899-f3d9b0a2974f.mp4

### Version

2022.1101.0-lazer

### Logs

[runtime.log](https://github.com/ppy/osu/files/9914848/runtime.log)
[network.log](https://github.com/ppy/osu/files/9914849/network.log)
[performance.log](https://github.com/ppy/osu/files/9914850/performance.log)
[input.log](https://github.com/ppy/osu/files/9914851/input.log)
[database.log](https://github.com/ppy/osu/files/9914852/database.log)
[performance-update.log](https://github.com/ppy/osu/files/9914853/performance-update.log)
[performance-draw.log](https://github.com/ppy/osu/files/9914854/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9914855/performance-input.log)
[performance-audio.log](https://github.com/ppy/osu/files/9914856/performance-audio.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9914857/legacy-ipc.log)

**Comment by peppy** [comment, marked as answer]
This is going to be overhead induced by your system. It does not happen on windows or macOS (or linux for others).

Please continue your investigation into your system configuration.
